### DISCUSSION TITLE
Using Mongo or Postgres

### DISCUSSION THREAD

**Discussion by nonfungibletunji** [BinaryBirds/feather#40] [Q&A]
`try feather.configure(database: .mongo(.file("db.sqlite")),
                      databaseId: .mongo,
                      fileStorage: .local(publicUrl: Application.baseUrl, publicPath: Application.Paths.public, workDirectory: "assets"),
                      fileStorageId: .local,
                      modules: [
                        FileBuilder(),
                        RedirectBuilder(),
                        BlogBuilder(),
                        AnalyticsBuilder(),
//                        AggregatorBuilder(),
                        SponsorBuilder(),
                        SwiftyBuilder(),
                        MarkdownBuilder(),
                      ])`

how does one change the configure, to use DB URL for either Mongo or Postgress? feather.configure documentation ??

**Comment by tib** [comment]
Hey, you should be able to use the MongoDB configuration just like any other Fluent driver.

https://docs.vapor.codes/4.0/fluent/overview/#mongodb

Also check this: 

https://github.com/FeatherCMS/feather/wiki/Project-structure

**Comment by nonfungibletunji** [comment]
Yeah I set it up just like the vapor tutorial docs. I will look more into it and maybe do a fresh install

**Reply by tib** [reply]
I just released a new beta version with a brand new config API for Feather, it'll more easy to switch to the Mongo driver. 

You should take a look at the new Sources/Drivers folder + main.swift file to see how it works.

**Reply by tib** [reply]
Also please check the README.md file, there are some new dotenv variable docs about how to config Postgres. ;)
